okhttp/src/main/kotlin/okhttp3/internal/Util.kt

```diff
@@ -670,4 +670,4 @@ internal inline fun <T> Iterable<T>.filterList(predicate: T.() -> Boolean): List
   return result
 }
 
-const val userAgent: String = "okhttp/${OkHttp.VERSION}"
+val userAgent: String = "okhttp/${OkHttp.VERSION}"
```

Comment:
This one should be a const though. Reference `$projectVersion` directly?

Reply:
By templating more code? I don't think that's worth the tiny benefit. If you've got a different idea I'm game.

Reply:
Generate the file with the non-const and const rather than templating:

https://github.com/cashapp/zipline/blob/85116a14c55dd2ed28138e1f483b43224943978f/zipline/build.gradle.kts#L17-L37
https://github.com/cashapp/zipline/blob/85116a14c55dd2ed28138e1f483b43224943978f/zipline/build.gradle.kts#L84

Reply:
Yeah, or generate a templated file with two symbols, one const and one not. Is const actually useful?

We save one string concatenation and pay for it with more code loaded into the heap for the lifetime of the process.